output.go

```diff
@@ -294,8 +294,8 @@ func (outputIDs OutputIDs) OrderedSet(set OutputSet) Outputs[Output] {
 }
 
 var (
-	// ErrDepositAmountMustBeGreaterThanZero returned if the deposit amount of an output is less or equal zero.
-	ErrDepositAmountMustBeGreaterThanZero = ierrors.New("deposit amount must be greater than zero")
+	// ErrAmountMustBeGreaterThanZero returned if the base token amount of an output is less or equal zero.
+	ErrAmountMustBeGreaterThanZero = ierrors.New("base token amount must be greater than zero")
```

Comment:
Nit: Make the error messages consistent. Write either "amount" or "base token amount" in both.